### ISSUE TITLE
How to prompt a dialog box of "Save As", when a user clicks a button?

### ISSUE THREAD

**Issue by sameerkumar18** [jiahaog/nativefier#377]
### Description
Currently, Nativefier is saving the file directly to the Downloads folder. I'm using Mac (darwin).
By dialog box, I mean the same "Save As" dialog box as on the Web when we right click in a browser.

### Specifications
- Version of Nativefier (run `$ nativefier --version`): `v7.3.1`
- Version of Node.js (run `$ node --version`): `v4.5.0`
- OS: `Mac`
- Architecture `OSX Darwin`
- Stack trace from the error message (if any)

```
<Stack trace here>
```

**Comment by ronjouch** [comment]
Hey @sameerkumar18, thanks for the feedback! Nativefier has no such "Save as" feature for now, sorry. More generally, the current download experience totally sucks, we know about it and track it in https://github.com/jiahaog/nativefier/issues/335 . I'm enriching the issue with your proposal of a "Save As" context menu feature enabling selection of destination folder 👍.

Closing this bug, do subscribe to https://github.com/jiahaog/nativefier/issues/335 if you want to follow the discussion.

**Comment by sameerkumar18** [comment]
Copying from what I wrote in #335 
Hey
I just found a way to enable "Save As" (not sure if this is already mentioned before).
Earlier, my web app was by default downloading the file straight to the download folder.

So basically, I just opened up the source files and edited the "main.js".
In main.js, I changed this if (!opts.saveAs) { item.setSavePath(filePath); } to this if (opts.saveAs) { item.setSavePath(filePath); } Removed the '!'
This worked absolutely fine, as now the "Save As" dialog box poped whenever there was a download event.

Nativefier 7.3.1, Node.js 4.5.0, macOS
